about.css

```diff
@@ -405,7 +404,7 @@ footer {
     /* width: 50%; */
     color: #272a31;
     font-size: 18px;
-    width: 15rem;
+    /* width: 15rem; */
   }
```

Comment:
- [ ] Please consider removing this and all other `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.